aggregate_spatial.json

```diff
@@ -0,0 +1,80 @@
+    "categories": [
+        "cubes",
+        "aggregate & resample"
+    ],
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A raster data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "raster-cube"
+            }
+        },
+        {
+            "name": "geometries",
+            "description": "Geometries as GeoJSON on which the aggregation will be based.",
+            "schema": {
+                "type": "object",
+                "subtype": "geojson"
+            }
+        },
+        {
+            "name": "reducer",
+            "description": "A reducer to be applied on all values of each geometry. A reducer is a single process such as ``mean()`` or a set of processes, which computes a single value for a list of values, see the category 'reducer' for such processes.",
+            "schema": {
+                "type": "object",
+                "subtype": "process-graph",
+                "parameters": [
+                    {
+                        "name": "data",
+                        "description": "An array with elements of any type.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "description": "Any data type."
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "name": "target_dimension",
+            "description": "The new dimension name to be used for storing the results. Defaults to `result`.",
+            "schema": {
+                "type": "string"
+            },
+            "default": "result",
+            "optional": true
+        }
+    ],
+    "returns": {
+        "description": "A vector data cube with the computed results. The vector data cube implicitly gets restricted to the bounds of the geometries as if ``filter_spatial()`` would have been used with the same values for the corresponding parameters immediately before this process.\n\nThe computed value is stored in dimension with the name that was specified in the parameter `target_dimension`.\n\nThe computation also stores information about the total count of pixels (valid + invalid pixels) and the number of valid pixels (see ``is_valid()``) for each geometry. These values are stored as new dimension with a dimension name derived from `target_dimension` by adding the suffix `_meta`. The new dimension has the dimension labels `total_count` and `valid_count`.",
+        "schema": {
+            "type": "object",
+            "subtype": "vector-cube"
+        }
+    },
+    "exceptions": {
+        "TooManyDimensions": {
+            "message": "The number of dimensions must be reduced to three for 'aggregate_spatial'."
```

Comment:
Would be better to be more specific that this refers to the raster datacube, not the vector one.

Reply:
There's only a single data cube this could refer to, the one passed in data. I guess it is logical that this doesn't refer to the cube that is returned as that's nothing you could influence as a user.

Reply:
But you do also pass the geometry, and that's something the user can influence. But I guess most of the times vectors wouldn't have more than three dimensions, but then again it depends on the vector data cube definition...

Reply:
We dont support 4+ dimensional GeoJSON thus it won't be a problem, I hope. We may revisit this when reintroducing vector data cubes to that parameter.

Reply:
That's fine, but I don't see any downsides to making the message more specific and read `The number of raster data cube dimensions must be reduced to three for 'aggregate_spatial'.` right now rather than doing it later.